backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/ProcessStepName.java

```diff
@@ -0,0 +1,37 @@
+package de.bund.digitalservice.ris.caselaw.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public enum ProcessStepName {
```

Comment:
Die Liste der Steps zu kennen und in einem Enum zu haben macht Sinn, wenn wir fachliche Unterschiede zwischen den Steps entwickeln wollen (z.B. "wenn eine Entscheidung in der QS ist, kann sie nicht veröffentlicht werden"). Ist sowas vorgesehen? Ansonsten könnten wir auch die Bezeichnungen aus der Datenbank als String weitergeben 🤔

Reply:
(Der Nachteil an dem Enum ist, dass er sehr eng an das SQL Script gekoppelt ist, was die Process Steps in die Datenbank schreibt)